docs/gettingstarted/index.rst

```diff
@@ -95,7 +95,7 @@ Getting started with Flocker
 
          .. code-block:: console
 
-            you@laptop:~$ sudo apt-get update && apt-get install -y gcc python2.7 python-virtualenv python2.7-dev && \
+            you@laptop:~$ sudo apt-get update && sudo apt-get install -y gcc python2.7 python-virtualenv python2.7-dev && \
```

Comment:
`yum` updates repository data automatically.  `apt-get` doesn't.  If any of the package versions have changed since you ran `apt-get update` last then you might get an error from the `apt-get install` command.